### PR TITLE
Ask user for account ID during "Sign in with Ledger"

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by Patrick1904** [comment]
@marcinbodnar what would be the best/easiest way to test this?

**Comment by marcinbodnar** [comment]
@Patrick1904 Best would be to test it locally, you can set `conts accountId = []` here so it will mimic a situation where Ledger is not able to find any account:
https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L469

**Review by mattlockyer — APPROVED** [review]
Tested e2e and works.
Future improvement: reduce complexity in `account.js` and/or `wallet.js`. Maybe move more ledger func to ledger actions?

File: src/utils/wallet.js [hunk 1]
```diff
@@ -753,6 +763,11 @@ class Wallet {
             }
         }
     }
+
+    dispatchShowLedgerModal(show) {
+        const [ action ] = store.getState().account.actionsPending.slice(-1)
+        store.dispatch(showLedgerModal({show, action}))
+    }
```
Comment (mattlockyer) [inline src/utils/wallet.js:770]: Any way to move this to ledger actions and import to wallet.js instead?

**Review by Patrick1904 — APPROVED** [review]
Tested locally and works as intended.

**Pull request merged**